tools/raml2html/README.md

```diff
@@ -4,15 +4,37 @@ Tool for generating static HTML from RAML definitions.
 
 ## Installation
 
-Install required dependencies before use. Go to raml2html root directory and run:
+PHP 7.4 and [Composer](https://getcomposer.org/) are required.
 
-```
-composer install
+To install required dependencies, go to raml2html directory and run:
+
+```sh
+composer install;
 ``` 
 
-To generate static HTML from RAML definitions, use the following code:
+## Usage
 
+Note: If PHP 7.2 is not the default PHP, please, adapt.
```

Comment:
```suggestion
Note: If PHP 7.4 is not the default PHP, please, adapt.
```
Do we need an example like "On a recent macOS, install php 7.4 (`brew tap exolnet/homebrew-deprecated; brew install php@7.4;`) then replace `php` with something like `/opt/homebrew/Cellar/php@7.4/7.4.33_1/bin/php` in the commands bellow."?

Reply:
Drop the whole note completely.

```suggestion
```

Reply:
Removed in 6585a76aa3d4813de24c6ba40bbde371b4c6fad1